### ISSUE TITLE
pip install does not work when using from python 3.11 environment 

### ISSUE THREAD

**Issue by mfischer-ucl** [isl-org/Open3D#6287]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### Describe the issue

pip install open3d does not resolve when using a python 3.11 environment. running it from a 3.8 environment works just fine. 

### Steps to reproduce the bug

```python
pip install open3d
```

### Open3D, Python and System information

```markdown
- Operating system: Fedora 36
- Python version: Python 3.8, 3.11
- System architecture: x86
- Is this a remote workstation?: no
- How did you install Open3D?: pip
```

**Comment by Omdiwan** [comment]
Same issue, I get an error when I try to install open3d via cmd
![image](https://github.com/isl-org/Open3D/assets/105396356/c4c2fe2c-2c1f-4233-bf5d-9de0176a83ce)

**Comment by johnthagen** [comment]
Please follow/ 👍: 

- #5620

Open3D does not yet support Python 3.11 and as such has not published 3.11 wheels onto PyPI.

**Comment by ssheorey** [comment]
Thanks to [OlivierLDff](https://github.com/OlivierLDff) who is working on Python 3.11 support in this PR: https://github.com/isl-org/Open3D/pull/6288

**Comment by johnthagen** [comment]
3.11 wheels have been published with 0.18:

- https://github.com/isl-org/Open3D/pull/6288#issuecomment-1878030429

This issue can be closed.
